### ISSUE TITLE
Unable to specify ListView description

### ISSUE THREAD

**Issue by famod** [jenkinsci/configuration-as-code-plugin#888]
Specifying a `ListView` description via YAML like:

```yaml
jenkins:
  views:
    - list:
        name: "test"
        description: "test desc"
```

fails with exception `io.jenkins.plugins.casc.ConfiguratorException: Invalid configuration elements for type class hudson.model.ListView : description.`

A view description that is entered manually via the UI also doesn't show up in the JCasC export.

Not sure what is going on here as `View` (the superclass of `ListView`) [does provide the respective getter](https://github.com/jenkinsci/jenkins/blob/jenkins-2.164.3/core/src/main/java/hudson/model/View.java#L291).

- Jenkins version: 2.164.3
-  Plugin version: 1.15

**Comment by casz** [comment]
Duplicate of #671

**Comment by casz** [comment]
https://issues.jenkins-ci.org/browse/JENKINS-57121

**Comment by casz** [comment]
Actually reported as https://issues.jenkins-ci.org/browse/JENKINS-57567

**Comment by famod** [comment]
Ah, thanks for the pointer. I did find JENKINS-57121 before, but thought it was not related.

Voted for JENKINS-57567.

**Comment by timja** [comment]
Fixed in jenkins core, Released in 2.180 of jenkins
